hw/top_earlgrey/ip/ast/data/ast_cdc_abstract.sgdc

```diff
@@ -0,0 +1,631 @@
+################################################################################
```

Comment:
@jdonjdon can you have a look at this and see if there's any easy way for us to re-use this in our cdc flow?

Reply:
@tjaychen  are you asking me or @jeoongp ?

Reply:
i'm sorry auto-completed incorrectly.
@jeoongp it is.

Reply:
VerixCDC support team said the tool does not support conversion from spy_glass model due to copyright issue but suggested an auto-mesh model based on black-box. They asked If we need to support detailed functions in CDC and consider this auto-mesh model if not. I asked again any easy way to re-use SGDC files in their cdc flow. I will keep you posted.

Reply:
yeah the ast model used in our sims / cdc is not the "real" one. So we definitely don't need a detailed cdc model.
From my perspective we just care that the ports are associated to the right domain, then we can do an integration check with the rest of the design. 

@Jacob-Levy / @arnonsha would you guys be open to generating a model using the tool? (ie, not spyglass?) Because otherwise this will be somewhat manual, and things could easily get out of sync as updates are made. 

@msfschaffner Adding Michael to the discussion as well.

Reply:
I agree a manual translation is needed, although it shouldn't be hard to do. Unfortunately we have no access to VerixCDC.
@tjaychen, as you said the top level run should treat AST as a black box, and check that the assumptions on the ports (coming from the translated SG CDC file) are indeed met.